telemetry/server.go

```diff
@@ -150,6 +151,39 @@ func (s *Server) updateEnvelope(w http.ResponseWriter, r *http.Request) {
 	)
 }
 
+func (s *Server) createSentEnvelope(w http.ResponseWriter, r *http.Request) {
+	start := time.Now()
+	var sentEnvelope SentEnvelope
+	decoder := json.NewDecoder(r.Body)
+	if err := decoder.Decode(&sentEnvelope); err != nil {
+		log.Println(err)
+
+		err := respondWithError(w, http.StatusBadRequest, "Invalid request payload")
+		if err != nil {
+			log.Println(err)
+		}
+		return
+	}
+	defer r.Body.Close()
+
+	err := sentEnvelope.put(s.DB)
+	if err != nil {
+		log.Println("could not save envelope", err, sentEnvelope)
+	}
```

Comment:
We should probably respond with a 500/503 error:
```suggestion
	if err != nil {
		log.Println("could not save envelope", err, sentEnvelope)
		err := respondWithError(w, http.StatusBadRequest, "could not save envelope")
		if err != nil {
			log.Println(err)
		}
		return
	}
```

Reply:
added in latest commit